src/Evolution/Systems/GeneralizedHarmonic/TimeDerivative.cpp

```diff
@@ -56,7 +56,6 @@ void TimeDerivative<Dim>::apply(
         christoffel_first_kind_3_up,
     const gsl::not_null<Scalar<DataVector>*> lapse,
     const gsl::not_null<tnsr::I<DataVector, Dim>*> shift,
-    const gsl::not_null<tnsr::ii<DataVector, Dim>*> spatial_metric,
```

Comment:
Does it make more sense to just make the spatial metric a reference tag then? Do we ever set it to something other than the spatial part of the spacetime metric? If it's a reference tag then you can take it as an argument rather than `not_null`. Then you don't have to make a view every time this is used.

Reply:
I don't think a reference tag works because those point at an entire different object, whereas this points to a subset. I _think_ compute tags can set the `DataVector`s into the spacetime metric, but adding the argument tags in the places is not something I can reasonably prioritize right now. I do agree that it's worth trying to do what you suggested in the long term, though!